source/concepts/applications.rst

```diff
@@ -208,18 +152,19 @@ If they are available, a client can choose to represent them as they choose.
 
```

Comment:
In the sentence above, what does "if they are available" mean? If what are available?

Reply:
I've removed this since it doesn't really communicate anything non-obvious ("if play area information is available, a client can display it how they wish")